File: pyhton64/nivel1/exer5.py
tempo_viagem = int(input('Tempo de viagem: '))
velocidade_viagem = int(input('Velocidade: '))
distancia = tempo_viagem * velocidade_viagem
litros_usados = distancia / 12
print(f'Litros usados: {litros_usados}')

File: java64/nivel1/exer5.java
import javax.swing.JOptionPane;

public class exer5 {
    public static void main(String[] args) {
        int tempo;
        int velocidade;
        int ditancia;
        double litros;

        tempo = Integer.parseInt(JOptionPane.showInputDialog(null, "Tempo: "));
        velocidade  = Integer.parseInt(JOptionPane.showInputDialog(null, "Velocidade: "));

        ditancia = tempo * velocidade;
        litros = ditancia / 12;

        JOptionPane.showMessageDialog(null, "Litros usados: " + litros);
    }
}